### ISSUE TITLE
Config inject should work directly on images, not just drives

### ISSUE THREAD

**Issue by pimterry** [balena-io/balena-cli#1003]
<img src="https://frontapp.com/assets/img/icons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_em7f)

**Comment by pdcastro** [comment]
Even more importantly, if it does not work on images, it should not document that it does! -
```
$ balena help config inject
...
OPTIONS
  -d, --drive <drive>  device filesystem or OS image location
```

**Comment by pdcastro** [comment]
Fixed/implemented in CLI v12.44.9. (Might have been implemented before, but the CLI was also calling `umount` on unmounted image files which would cause errors, and this was fixed in #2251.)
